web_gestures/static/src/js/gestures.js

```diff
@@ -0,0 +1,29 @@
+odoo.define('web_gestures', function (require) {
```

Comment:
The whole module should be named in singular, please rename it to ```web_gesture```

Reply:
done